lib/community.js

```diff
@@ -527,7 +527,15 @@ exports.all = {
         width: 150,
         quote: 'Hapi allows us to quickly build web-applications, safe in the knowledge that it is feature-complete, battle-tested and production ready. Its plugin system lets us easily break out parts of the system as we scale our dev team. Hapi gets out of the way, so we can get on with building business logic.',
         person: 'Antony Jones, Chief Technical Architect'
+    },
+    'voxgig': {
```

Comment:
Would be nice to capitalize the name of the company as all the others. Also the list is kept in alphabetical order so it would be nice to keep it that way.